<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>TeacherServer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="videoViewer">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1024</width>
      <height>576</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_camera">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>660</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Camera</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_screen">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>610</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Screen</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cb_camera">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>660</y>
      <width>390</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_audio">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>710</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Audio</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cb_device">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>710</y>
      <width>390</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cb_camRes">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>660</y>
      <width>390</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="slider_volume">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>710</y>
      <width>270</width>
      <height>30</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="pageStep">
     <number>5</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::NoTicks</enum>
    </property>
   </widget>
   <widget class="QLabel" name="volume">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>710</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Volume: 100</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_screenPen">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>610</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>ScreenPen</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cb_screenRes">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>610</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_fileTrans">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>610</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>FileTrans</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_signIn">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>610</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>SignIn</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cb_screenHz">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>610</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_textChat">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>610</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>TextChat</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
